x-element.js

```diff
@@ -1042,9 +1087,9 @@ class TemplateEngine {
    * @returns {any}
    */
   static html(strings, ...values) {
-    const symbol = Symbol();
-    const result = TemplateEngine.#createResult('html', strings, values);
-    TemplateEngine.#resultMap.set(symbol, result);
+    const symbol = Object.create(null);
```

Comment:
Turns out… (1) FF still doesn’t support Symbols as WeakMap keys and (2) it’s more performant to call `Object.create(null)` for this purpose — theperfdontlie.